http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/DiskRoutes.scala

```diff
@@ -190,10 +190,8 @@ object DiskRoutes {
   implicit val updateDiskRequestDecoder: Decoder[UpdateDiskRequest] = Decoder.instance { x =>
     for {
       l <- x.downField("labels").as[LabelMap]
-      us <- x.downField("updateSize").as[Option[DiskSize]]
-      ud <- x.downField("updateDiskType").as[Option[DiskType]]
-      ub <- x.downField("updateBlockSize").as[Option[BlockSize]]
-    } yield UpdateDiskRequest(l, us, ud, ub)
+      us <- x.downField("updateSize").as[DiskSize]
```

Comment:
we should just call this `size`...being an `UpdateDiskRequest` already indicates this is for updating the disk size. terra UI preferred the field matches other requests so that they don't have to adjust the field name differently depending on the call they're making

Reply:
will update :)